flow/project.py

```diff
@@ -709,11 +713,15 @@ class FlowGroupEntry(object):
         commands to execute.
     :type options:
         str
+    :param aggregator:
+        aggregator object associated with the :py:class:`FlowGroup`
+    :type aggregator:
+        :py:class:`aggregator`
     """
-
-    def __init__(self, name, options=""):
+    def __init__(self, name, options="", aggregator=aggregator.groupsof(1)):
```

Comment:
@vyasr Have we decided that groups will determine the aggregation rather than operations?

Reply:
@b-butler If I understand your question correctly, I believe that is what we decided.

Reply:
The implication of this choice is that all operations in a group must have the same aggregator. I think that makes sense, yes.